client/src/Components/Cards.js

```diff
@@ -9,7 +9,7 @@ import "./Cards.css"
 
 export const Cards = () => (
     <>
-        <CardDeck className="mt-5 text-center">
+        <CardDeck className="text-center alignCard">
```

Comment:
So I see you added a new class here with the name `alignCard` but I'm wondering if there might be a more descriptive name we could use. For example, what is it that this class is doing? Is that something that we could add to the name so that it's clear what alignment is being done in this CSS class? Just reading the class name, it's a little more difficult to understand exactly what it's supposed to be used for.